build.sh

```diff
@@ -17,10 +17,6 @@
 # Package build scripts can use this by setting $DESTDIR before launching the
```

Comment:
Should we add `set -x` here too like in the other scripts? How about adding `-e` as well?

Reply:
Done. Added -x to the other scripts as well